magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/LSTnsb.py

```diff
@@ -65,12 +92,26 @@ def nsb(run_list, simtel, lst_config, run_number, denominator):
         if ii % mod == 0:
             try:
                 a, _, _ = calculate_noise_parameters(simtel, run_list[ii], lst_config)
-                noise.append(a)
-                logger.info(a)
+                if a is not None:
+                    if a > 0.0:

```

Comment:
I do not think a negative output of calculate_noise_parameters is possible, see here:

https://github.com/cta-observatory/cta-lstchain/blob/378a4c3a8a1e72d8e3e1b6739edef9e591002261/lstchain/image/modifier.py#L393-L409

you get positive value if there is noise to be added, and if noise in data looks smaller than in MC, it is clipped to 0, but this is not a problem, just a very "dark" run. 
have you seen any cases with negative values?

Reply:
thx for checking it as we discussed, resolving comment